content/en/platform/corda/4.11/community/api-flows.md

```diff
@@ -1362,8 +1381,7 @@ Threading must be explicitly controlled when using `FlowExternalAsyncOperation`.
 
 Implementations of `FlowExternalAsyncOperation` must return a `CompletableFuture`. The developer decides how to create this future.
 The best practice is to use `CompletableFuture.supplyAsync` and supply an executor to run the future. You can use other libraries to
-generate futures, as long as a `CompletableFuture` is returned out of `FlowExternalAsyncOperation`. You can see an example of creating a future
-using Guava’s ListenableFuture below.
+generate futures, as long as a `CompletableFuture` is returned out of `FlowExternalAsyncOperation`. You can see an example of creating a future using [Guava’s ListenableFuture](#api-flows-guava-future-conversion) Below.
```

Comment:
```suggestion
generate futures, as long as a `CompletableFuture` is returned out of `FlowExternalAsyncOperation`. An example of creating a future using [Guava’s ListenableFuture](#api-flows-guava-future-conversion) is shown below.
```

Can we have a cross-ref or something rather than using "below"?

Reply:
Added anchors to both Ent and Community so link works.